sys/dev/sound/usb/uaudio.c

```diff
@@ -209,7 +209,7 @@ struct uaudio_mixer_node {
 };
 
 struct uaudio_configure_msg {
-	struct usb_proc_msg hdr;
+	struct usb_proc_msg hdr __subobject_member_used_for_c_inheritance;
```

Comment:
Would be nice if we could annotate the type instead so that we don't need all those changes... But until someone does that compiler work this is what we're stuck with.